konfera/event/views.py

```diff
@@ -53,30 +55,65 @@ def event_details_view(request, slug):
     return render(request=request, template_name='konfera/event/details_conference.html', context=context)
 
 
-def cfp_form_view(request, slug):
-    event = get_object_or_404(Event.objects.published(), slug=slug)
-    context = dict()
-    speaker_form = SpeakerForm(request.POST or None, prefix='speaker')
-    talk_form = TalkForm(request.POST or None, prefix='talk')
+class CFPView(TemplateView):
+    template_name = 'konfera/cfp_form.html'
+    message_text = _("Your talk proposal was successfully created.")
 
-    if speaker_form.is_valid() and talk_form.is_valid():
-        speaker_instance = speaker_form.save()
-        talk_instance = talk_form.save(commit=False)
-        talk_instance.primary_speaker = speaker_instance
-        talk_instance.status = Talk.CFP
-        talk_instance.event = Event.objects.get(slug=slug)
-        talk_instance.save()
-        message_text = _("Your talk proposal successfully created.")
-        messages.success(request, message_text)
+    def dispatch(self, *args, **kwargs):
+        get_object_or_404(Event, slug=kwargs.get('slug'))
```

Comment:
above or bellow this (`get_object_or_404(Event, slug=kwargs.get('slug'))`) line?